docs/Apps/Data Studio/CCLE Correlation Analysis.md

```diff
@@ -1,11 +1,235 @@
+
+*   **Select Column:** Names of the columns from the metadata file are enlisted here for you to choose.
+
+*   **Column Value:** Once a column has been selected in the *Select Column* dropdown, then all the unique values from the selected column are available for you to pick from.
+
+Once all the fields are selected, click on *Run Task* to execute the component.
+
+It will generate three output files:
+
+![Filtering output](../../img/ccle/10.png) <center>**Figure 10.** Filtering output</center>
+
+*   **Expression data:** Subsetted expression data with filtered samples.
+
+*   **Mutation data:** Subsetted mutation matrix with filtered samples.
+
+*   **Metadata:** Subsetted metadata with filtered samples.
+   
+
+# Data Exploration
+
+## Gene expression Box plot
+
+This component generated a gene expression Boxplot. It is a visualization component that uses a five-number summary ( minimum, first quartile, median, third quartile, and maximum) to display the distribution of data based on quartiles. Outliers are also displayed as individual points.
+
+![Box plot interface](../../img/ccle/11.png) <center>**Figure 11.** Boxplot parameters</center>
+
+*   **Select gene:** Dropdown for picking the gene for which the boxplot is to be made.
+
+*   **Select metadata column:** Dropdown to select one of the metadata columns.  
+
+After executing the component, an interactive *Boxplot* will be generated as an output.
+
+![Box plot](../../img/ccle/12.png) <center>**Figure 12.** Boxplot</center>
+    
+
+## **PCA**
+
+This component allows you to simplifies the complexity of high-dimensional data while retaining the trends and patterns in it. It projects the data onto a lower dimension with an objective to find the best summary of the data using a limited number of principal components that help in understanding the clustering pattern between biologically grouped and ungrouped samples.
+
+![PCA](../../img/ccle/13.png) <center>**Figure 13.** PCA parameters</center>
+
+*   **Cohort Column:** Dropdown to select one of the metadata columns.
+
+*   **Top N Variants:** The top N variable entities will be used for PCA calculation. Define the number in this box. The default number used is 1500.
+
+It generated two outputs:
+
+*   **PCA Plot:** A plot is created where the samples are labeled based on the cohort selected in the metadata column. When the user hovers over the points, sample ID and percentage of variance explained by each principal component are displayed along with the cohort.
+
+*   **PCA Score:** Table of the first 10 PC values and metadata columns.  
+
+![PCA](../../img/ccle/14.png) <center>**Figure 14.** PCA outputs</center>
+   
+
+## **Correlation**
+
+This component uses Fisher's exact test to test for correlation between the mutation status of the selected gene vs the other selected features.
```

Comment:
against other selected features. consistent with what is written in scope of the app